### PR TITLE
Few more framework level steps

### PR DESCRIPTION
**Pull request by rajesh-pandian**

### CODE REVIEW

File: lib/page_steps.js [hunk 1]
```diff
@@ -71,3 +71,249 @@ step(
...
+  const propertyValue = process.env[text];
+  await taiko.write(propertyValue, taiko.into(taiko.textBox(textbox)));
+});
+
+step("Write <text> from spec memory into <textbox>", async (text, textbox) => {
+  const propertyValue = gauge.dataStore.specStore.get(text);
+  await taiko.write(propertyValue, taiko.into(taiko.textBox(textbox)));
+});
+
+step("wait <milliseconds> milliseconds", async (milliseconds) => {
```
Comment (ramsbaron) [inline lib/page_steps.js:90]: To be consistent I propose we start the step with uppercase

File: lib/page_steps.js [hunk 2]
```diff
@@ -71,3 +71,249 @@ step(
...
+  await taiko.waitFor(milliseconds);
+});
+
+step("wait for <text>", async (text) => {
```
Comment (ramsbaron) [inline lib/page_steps.js:94]: same here, propose uppercasing the steps

**Review by ramsbaron — CHANGES_REQUESTED** [review]

File: lib/page_steps.js (hunk 1 above)
Comment (mkmurali) [reply lib/page_steps.js:90]: That's a good suggestion @ramsbaron. Each step is read like a sentence and capitalizing it makes it easy to read.

**Review by mkmurali — CHANGES_REQUESTED** [review]
Could you please also update the README file with these new steps.

**Comment by mkmurali** [comment]
Pull request #8 included all these changes and suggestions.

**Pull request closed without merging**